builtin/aws/ecs/releaser.go

```diff
@@ -51,107 +62,90 @@ func (r *Releaser) Release(
 	}
 	elbsrv := elbv2.New(sess)
 
-	dlb, err := elbsrv.DescribeLoadBalancers(&elbv2.DescribeLoadBalancersInput{
```

Comment:
Here's a part of the old bug - if the user specified a listener in the deploy config, we would always try to describe the load balancer here. The deploy plugin didn't set the load balancer arn, so this would be empty string and we'd get a 400.